QuestionAnswering/QA_ICL/frontend/next_app_marie/src/components/layout/footer.tsx

```diff
@@ -6,6 +6,7 @@ import { cn } from '@/lib/utils'
 import CMCLLogo from '@/public/images/cmcl.svg'
 import COMOLogo from '@/public/images/como_logo_processed.svg'
 import CARESLogo from '@/public/images/cares_short_logo_processed.svg'
+import MITLogo from '@/public/images/MIT-logo-transparent-cropped.png'
```

Comment:
Please replace this with the `.svg` version, and please do add the file to the repository.

Reply:
Addressed in commit 749333f8bf21160b303b92d54c991db89fbac08a.